### PR TITLE
Implement GetUserTokens lookup function

### PR DESCRIPTION
**Pull request by sebinjohn**
- [x] All [tests](https://github.com/jfrog/jfrog-client-go#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----

Implements two changes
1. Change the anonymous struct inside GetTokensResponseData.
2. Implement a GetUserTokens lookup function to lookup tokens issued to a username.

I would like to implement some tests for the second item, but require some help around what is the best way to write a test case for function which doesn't take *Params as args.

### CODE REVIEW

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
Thanks for adding this @sebinjohn.
Let's add a test for this and also document the new API in the README.

**Comment by sebinjohn** [comment]
> Thanks for adding this @sebinjohn.
> Let's add a test for this and also document the new API in the README.

@eyalbe4 I would like to implement some tests for the second item, but require some help around what is the best way to write a test case for function which doesn't take *Params as args.

**Comment by eyalbe4** [comment]
@sebinjohn, I think you can take a similar approach to the one taken in ```getTokensTest```,

**Comment by sebinjohn** [comment]
@eyalbe4  I have implemented a test case.

File: tests/artifactorysecurity_test.go [hunk 1]
```diff
@@ -136,6 +136,42 @@ func getTokensTest(t *testing.T) {
 	tokensToRevoke = append(tokensToRevoke, token.RefreshToken)
 }
 
+func getUserTokenstest(t *testing.T) {
```
Comment (eyalbe4) [inline tests/artifactorysecurity_test.go:139]: Let's make sure that the test revokes the tokens it generates. See the ```tokensToRevoke``` slice used in other tokens tests.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: tests/artifactorysecurity_test.go (hunk 1 above)
Comment (sebinjohn) [reply tests/artifactorysecurity_test.go:139]: Done.

**Pull request merged**